nvflare/private/fed/client/communicator.py

```diff
@@ -266,8 +267,9 @@ def submit_update(
         )
         end_time = time.time()
         return_code = result.get_header(MessageHeaderKey.RETURN_CODE)
+        size = task_message.get_header(MessageHeaderKey.PAYLOAD_LEN)
```

Comment:
Here the size calculation is not correct. It always got 2. It looks like the header not been set correctly. Could you please check?

Reply:
I think there is an error in the calculation - see my comment in core_cell.py.

Reply:
Fixed.